src/uct/tcp/tcp_cm.c

```diff
@@ -545,31 +545,25 @@ unsigned uct_tcp_cm_conn_progress(uct_tcp_ep_t *ep)
 
 ucs_status_t uct_tcp_cm_conn_start(uct_tcp_ep_t *ep)
 {
-    uct_tcp_ep_conn_state_t new_conn_state;
-    uint32_t req_events;
     ucs_status_t status;
 
     status = ucs_socket_connect(ep->fd, (const struct sockaddr*)&ep->peer_addr);
-    if (status == UCS_INPROGRESS) {
-        new_conn_state  = UCT_TCP_EP_CONN_STATE_CONNECTING;
-        req_events      = UCS_EVENT_SET_EVWRITE;
-        status          = UCS_OK;
+    if (UCS_PTR_IS_ERR(status)) {
```

Comment:
status is not a pointer, need to use `UCS_STATUS_IS_ERR`

Reply:
done